lib/dialects/postgres/schema/columncompiler.js

```diff
@@ -8,6 +8,7 @@ const { isObject } = require('lodash');
 function ColumnCompiler_PG() {
   ColumnCompiler.apply(this, arguments);
   this.modifiers = ['nullable', 'defaultTo', 'comment'];
+  this._nativeEnumCreated = false;
```

Comment:
Are you sure this works, when there are multiple native enums declared / altered in the same table? This is a column compiler, so it probably does, but anyways set of tests for this would be needed. Also other tests than just having the most trivial happy path tested (cases where multiple columns are created and some altered)

Reply:
Added some tests